src/django/pfb_analysis/models.py

```diff
@@ -240,6 +240,10 @@ def census_blocks_url(self):
         return self._s3_url_for_result_resource('neighborhood_census_blocks.zip')
 
     @property
+    def connected_census_blocks_url(self):
+        return self._s3_url_for_result_resource('neighborhood_connected_census_blocks.csv.zip')
```

Comment:
All this repetition of the list of exports is begging to be consolidated into a config object in settings.  Then these model and view pieces could loop over it and the analysis could get it as (yet another) environment variable.  Or something along those lines.  Not on this PR, though.  We should open an issue.